codes/dataops/common.py

```diff
@@ -1,152 +1,150 @@
     return env, paths
 
 
-###################### read images ######################
 def _read_lmdb_img(env, key, size=None):
-    '''read image from lmdb with key (w/ and w/o fixed size)
-    size: (C, H, W) tuple'''
+    """
+    Read image from lmdb with key (w/ and w/o fixed size).
+    size: (C, H, W) tuple.
+    """
     with env.begin(write=False) as txn:
         buf = txn.get(key.encode('ascii'))
         buf_meta = txn.get((key + '.meta').encode('ascii')).decode('ascii')
     img_flat = np.frombuffer(buf, dtype=np.uint8)
     if size:
-        C, H, W = size
+        c, h, w = size
     else:
-        H, W, C = [int(s) for s in buf_meta.split(',')]
-    img = img_flat.reshape(H, W, C)
-    return img
+        h, w, c = [int(s) for s in buf_meta.split(',')]
+    image = img_flat.reshape(h, w, c)
+    return image
 
 
 def read_img(env, path, out_nc=3, fix_channels=True, size=None):
-    '''
-        Reads image using cv2 (rawpy if dng), from lmdb bor from a buffer 
-        (path=buffer). Could also use using PIL instead of cv2.
-    Arguments:
-        path: image path or buffer to read
-        out_nc: Desired number of channels
-        fix_channels: changes the images to the desired number of channels
-        size: fixed size to use for lmbg (optional)
-    Output:
-        Numpy HWC, BGR, [0,255] by default 
-    '''
-
-    img = None
-    if env is None or env is 'img':  # img
-        if(path[-3:].lower() == 'dng'): # if image is a DNG
+    """
+    Reads image using cv2 (rawpy if dng), from lmdb bor from a buffer (path=buffer).
+    Could also use PIL instead of cv2.
+    :param path: image path or buffer to read
+    :param out_nc: output number of channels
+    :param fix_channels: changes the images to the desired number of channels
+    :param size: fixed size to use for lmbg (optional)
+    :returns: Numpy HWC, BGR, [0,255] by default
+    """
+    image = None
+    if env is None or env is 'img':
+        if path[-3:].lower() == 'dng':  # if image is a DNG
             import rawpy
             with rawpy.imread(path) as raw:
-                img = raw.postprocess()
-        if(path[-3:].lower() == 'npy'): # if image is a NPY numpy array
+                image = raw.postprocess()
+        if path[-3:].lower() == 'npy':  # if image is a NPY numpy array
             with open(path, 'rb') as f:
-                img = np.load(f)
-        else: # else, if image can be read by cv2 
-            img = cv2.imread(path, cv2.IMREAD_UNCHANGED)
-        #TODO: add variable detecting if cv2 is not available and try PIL instead
+                image = np.load(f)
+        else:  # else, if image can be read by cv2
+            image = cv2.imread(path, cv2.IMREAD_UNCHANGED)
+        # TODO: add variable detecting if cv2 is not available and try PIL instead
         # elif: # using PIL instead of OpenCV
-            # img = Image.open(path).convert('RGB')
+        # img = Image.open(path).convert('RGB')
```

Comment:
Leave indentantion, when used will be inside the "elif"

Reply:
This was an error due to re-format. Done.